nef-models/MenuEditorCommand+URLQueryItem.swift

```diff
@@ -0,0 +1,20 @@
+//  Copyright © 2020 The nef Authors.
+
+import Foundation
+
+public extension MenuEditorCommand {
+    func item(code: String) -> URLQueryItem {
+        switch self {
+        case .exportSnippetToFile:
+            return URLQueryItem(name: key, value: code)
+        case .exportSnippetToClipboard:
+            return URLQueryItem(name: key, value: code)
+        case .markdownPage:
+            return URLQueryItem(name: key, value: code)
+        case .playgroundBook:
+            return URLQueryItem(name: key, value: code)
```

Comment:
These 4 could be grouped as a single case, as you do below with `preferences` and `about`.